managed-ledger/src/main/java/org/apache/bookkeeper/mledger/ManagedLedger.java

```diff
@@ -595,4 +594,10 @@ void asyncSetProperties(Map<String, String> properties, final AsyncCallbacks.Upd
      * Get the ManagedLedgerInterceptor for ManagedLedger.
      * */
     ManagedLedgerInterceptor getManagedLedgerInterceptor();
+
+    /**
+     * Get basic ledger summary after the ledger is closed.
+     * will got exception if corresponding ledger was not closed when the method called.
+     */
+    CompletableFuture<LedgerInfo> getClosedLedgerInfo(long ledgerId);
```

Comment:
> Is it better to named getLedgerInfo? and does not distinguish whether it is closed. The closed state should contains in the LedgerInfo(Now I think we use the entries=0 to determine the ledger close or not)

My design is refuse to return an unclosed result to prevent caller to get an incomplete result in accident.